win-odbc64/odbcclient/build_os.bat

```diff
@@ -108,78 +93,68 @@ REM msbuild.exe /t:rebuild Drvr35Trace_os.vcxproj /p:Platform=x64 /p:Configurati
 REM	goto Exit
 REM )
 
 echo Building TCPIPV4 - Win64 Release...
 cd %SRCDIR%\odbcclient\Drvr35\TCPIPV4
-msbuild.exe /t:rebuild TCPIPV4_os.vcxproj /p:Platform=x64 /p:Configuration=Release
+msbuild.exe /t:rebuild TCPIPV4_os.vcxproj /p:Platform=x64 /p:Configuration=Release /p:ZlibIncludeDir=%ZLIB_INCLUDE_PATH% /p:ZlibLibDir=%ZLIB_LIB_PATH%
 set BUILD_STATUS=%ERRORLEVEL%
 if %BUILD_STATUS%==0 (
 	echo Build TCPIPV4 success
-) else ( 
+) else (
 	echo Build TCPIPV4 failed
 	goto Exit
 )
 
 echo Building TCPIPV6 - Win64 Release...
 cd %SRCDIR%\odbcclient\Drvr35\TCPIPV6
-msbuild.exe /t:rebuild TCPIPV6_os.vcxproj /p:Platform=x64 /p:Configuration=Release
+msbuild.exe /t:rebuild TCPIPV6_os.vcxproj /p:Platform=x64 /p:Configuration=Release /p:ZlibIncludeDir=%ZLIB_INCLUDE_PATH% /p:ZlibLibDir=%ZLIB_LIB_PATH%
 set BUILD_STATUS=%ERRORLEVEL%
 if %BUILD_STATUS%==0 (
 	echo Build TCPIPV6 success
-) else ( 
+) else (
 	echo Build TCPIPV6 failed
 	goto Exit
 )
 
 echo Building TranslationDll - Win64 Release...
 cd %SRCDIR%\odbcclient\TranslationDll
-msbuild.exe /t:rebuild TranslationDll_os.vcxproj /p:Platform=x64 /p:Configuration=Release	
+msbuild.exe /t:rebuild TranslationDll_os.vcxproj /p:Platform=x64 /p:Configuration=Release
 set BUILD_STATUS=%ERRORLEVEL%
 if %BUILD_STATUS%==0 (
 	echo Build TranslationDll success
-) else ( 
+) else (
 	echo Build TranslationDll failed
 	goto Exit
 )
 
 echo Building Drvr35Res - Win64 Release...
-cd %SRCDIR%\odbcclient\Drvr35Res 
-msbuild.exe /t:rebuild Drvr35Res_os.vcxproj /p:Platform=x64 /p:Configuration=Release	
+cd %SRCDIR%\odbcclient\Drvr35Res
+msbuild.exe /t:rebuild Drvr35Res_os.vcxproj /p:Platform=x64 /p:Configuration=Release
 set BUILD_STATUS=%ERRORLEVEL%
 if %BUILD_STATUS%==0 (
 	echo Build Drvr35Res success
-) else ( 
+) else (
 	echo Build Drvr35Res failed
 	goto Exit
 )
-   
-echo Building SetCertificateDirReg InstallHelper - Win64 Release...
-cd %SRCDIR%\Install\SetCertificateDirReg\SetCertificateDirReg
-msbuild.exe /t:rebuild SetCertificateDirReg_os.vcxproj /p:Platform=x64 /p:Configuration=Release /p:Odbccp32lib64Dir=%ODBCCP32_64_PATH% 
-set BUILD_STATUS=%ERRORLEVEL%
-if %BUILD_STATUS%==0 (
-	echo Build SetCertificateDirReg success
-) else ( 
-	echo Build SetCertificateDirReg failed
-	goto Exit
-)
- 
-echo Building UpdateDSN InstallHelper - Win64 Release...
-cd %SRCDIR%\Install\UpdateDSN\UpdateDSN
-msbuild.exe /t:rebuild UpdateDSN_os.vcxproj /p:Platform=x64 /p:Configuration=Release
-set BUILD_STATUS=%ERRORLEVEL%
-if %BUILD_STATUS%==0 (
-	echo Build UpdateDSN success
-) else ( 
-	echo Build UpdateDSN failed
-	goto Exit
-)
+
 set ALL_SUCCESS=1
 
+cd %SRCDIR%\odbcclient
+copy /Y %BUILDDIR%\win-odbc64\odbcclient\inc\trafsqlext.h C:\Build\winodbc64\inc
+if not exist C:\Build\winodbc64\lib\x64\Release (
+	mkdir C:\Build\winodbc64\lib\x64\Release
+)
+copy /Y %BUILDDIR%\lib\x64\Release C:\Build\winodbc64\lib\x64\Release
+
+ISCC.exe /Q %BUILDDIR%\win-odbc64\Install\win64_installer\installer.iss

```

Comment:
Since installer script  is bundling c++ runtime redistributable library, this script will fail as the redist directory does not exist. My suggestion would be to create the c:\Build\winodbc64  folder as the first step and create the inc and lib folders. Then we don't need the c:\lib folder  to keep all the dlls